Celeste.Mod.mm/Mod/Everest/Logger.cs

```diff
@@ -10,6 +10,70 @@
 namespace Celeste.Mod {
     public static class Logger {
 
+        private static Dictionary<string, LogLevel> minimumLevels = new Dictionary<string, LogLevel>();
+        private static Dictionary<string, LogLevel> minimumLevelsFromEverestSettings = new Dictionary<string, LogLevel>();
+        private static Dictionary<string, LogLevel> minimumLevelsCache = new Dictionary<string, LogLevel>();
+
+        /// <summary>
+        /// Sets the minimum log level to be written in the logs for lines matching the given tag prefix.
+        /// When using this, make sure the tag prefix is restrictive enough not to impact other mods
+        /// (for example, if all your tags follow the format MyMod/xxx, use "MyMod/" as a prefix).
+        /// </summary>
+        /// <param name="tagPrefix">The prefix of the log tags to affect with this log level</param>
+        /// <param name="minimumLevel">The minimum level of logs to print out in the logs</param>
+        /// (useful to make levels set by code overridable through modsettings-Everest.celeste)</param>
```

Comment:
Please fix the xml tags in this comment, and consider adding some `<br/>` or `<para />` tags to the summary

Reply:
Done, this line was actually a leftover from an extra parameter that was removed during development.